collins/collins.go

```diff
@@ -159,14 +166,47 @@ func NewClientFromFiles(paths ...string) (*Client, error) {
 		Host     string
 		Username string
 		Password string
+		Timeout  int
```

Comment:
Why is this also not a `time.Duration`?

Reply:
The real reason was because I was just mapping this to the expected YAML values which for timeout would be an int. However I just messed around with using time.Duration instead and when unmarshal is called this this results in an int of 120 being 120ns. I don't know of anyway without writing a custom unmarshaler to make that more sane.

Reply:
Ah, yeah, that's a fair point. Let's not change yaml parsing behavior. My only real goal is to make sure that for `NewClient`, it's taking a `time.Duration` instead of an `int` for timeouts. Whatever happens in parsing I care significantly less about.